src/components/schedule-page/table-middleline.tsx

```diff
@@ -7,15 +7,16 @@ export interface TableMiddleLineOptions {
 
 export function TableMiddleLine({ name, children }: TableMiddleLineOptions): JSX.Element {
   return (
-    <tr className="middleSection">
-      <td className="middleCell">
-        <span>
-          {name} <LineMiddle component={children} />
-        </span>
-      </td>
-      <td colSpan={2}>
-        <hr className="middle" />
-      </td>
-    </tr>
+    <>
```

Comment:
To jest potrzebne ponieważ react oczekuje, że funkcja zawsze będzie zwracać jeden komponent. 
A bez `<></>`, wygląda tak, niby zwraca dwa:
1. `tr`
2. `LineMiddle`